internal/db/queries.sql

```diff
@@ -96,3 +96,8 @@ INSERT INTO mergestat.repo_sync_logs (repo_sync_queue_id, log_type, message)
 SELECT id, 'ERROR', 'No response from job within reasonable interval. Timing out.' FROM timed_out_sync_jobs
 RETURNING repo_sync_queue_id
 ;
+
+-- name: DeleteRemovedRepos :exec 
+DELETE FROM public.repos WHERE repo NOT IN (@currentRepos::text[])
```

Comment:
I started a review yesterday but cancelled it once I saw @patrickdevivo had started one too. The only comment I have is that I think we should include the context of the `repo_import_id` in the where clause here to only compare against the repos imported by that import. This will allow other imports to continue working as well as manually added repos.